src/components/movieDetails/movieDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import './movieDetails.css';
+
+interface MovieDetailsProps {
+  imageUrl: string;
+  movieName: string;
+  releaseYear: number;
+  rating: number;
+  duration: number;
+  description: string;
+}
+
+const MovieDetails = (props: MovieDetailsProps) => {
+  const formatDuration = () => {
+    const hours = Math.floor(props.duration / 60);
+    const remainingMinutes = props.duration % 60;
+    return `${hours ? `${hours}h` : ''}${
+      remainingMinutes ? ` ${remainingMinutes}min` : ''
+    }`;
+  };
+  return (
+    <div className="movieDetailsConntainer">
```

Comment:
Please pay ATTENTION to the classNames naming.
First of all - use kebab case.
And follow one convention trough the project. At the moment it is messy - each component with different approach. Hard to work with and debug this code. 
Read about BEM methodology (https://getbem.com/) - it will help you with naming.

The code you are writing is not for you - it is for some developer in future maintaining or improving the project. And it needs to be readable and understandable.  :)